tests/node-selector_test.go

```diff
@@ -0,0 +1,327 @@
+package tests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	"github.com/litmuschaos/litmus-e2e/pkg"
+	"github.com/litmuschaos/litmus-e2e/pkg/environment"
+	"github.com/litmuschaos/litmus-e2e/pkg/types"
+	"github.com/litmuschaos/litmus-e2e/utils"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/klog"
+)
+
+func TestGoNodeSelector(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = Describe("BDD of Node selector", func() {
+	Context("Test for target node selection by node name", func() {
+		It("Execute the node-cpu-hog experiment using a valid TARGET_NODES value", func() {
```

Comment:
From where are we getting this TARGET_NODES value?

Reply:
Added function for obtaining random node